src/RejectedPromise.php

```diff
@@ -63,6 +63,8 @@ public function wait($unwrap = true, $defaultDelivery = null)
         if ($unwrap) {
             throw Create::exceptionFor($this->reason);
         }
+
+        return null;
```

Comment:
This will fix 

> ERROR: InvalidReturnType - src/RejectedPromise.php:61:21 - No return statements were found for method GuzzleHttp\Promise\RejectedPromise::wait but return type 'mixed' was expected (see https://psalm.dev/011)
    public function wait($unwrap = true, $defaultDelivery = null)


"Return mixed" does not include `void`